sdg/components.py

```diff
@@ -330,7 +288,11 @@ def get_git_host(repo_url):
     http_client = httpx.Client(verify=ssl.create_default_context())
     client = openai.OpenAI(base_url=endpoint, api_key=api_key, http_client=http_client)
 
-    taxonomy_base = "main" if repo_branch or (repo_pr and int(repo_pr) > 0) else "empty"
+    taxonomy_base = (
```

Comment:
I'm not sure about this. See my comment [here](https://github.com/opendatahub-io/ilab-on-ocp/pull/298#discussion_r2025165958). 

If you end up merging it, I think it should be explained in the pipeline parameter description (and in the docs)

Reply:
Merged your PR and removed the commit here.